### ISSUE TITLE
typescript/jump-to-type-def fails when referencing node module

### ISSUE THREAD

**Issue by codejamninja** [syl20bnr/spacemacs#11628]
#### Description :octocat:

In the typescript major mode, when I try to run `typescript/jump-to-type-def` command while the
cursor is over a type imported from a npm module, nothing happens. Please note that this works
correctly when it is a type defined in my source code. It only fails when it is a type referencing
a node module.

#### Reproduction guide :beetle:
- Go to a typescript project and run `npm install`
- Start Emacs
- Go to a typescript file that uses a type from an npm module
- Move the cursor over a type imported from an npm module
- Run `SPC m g t` to run the `typescript/jump-to-type-def` command

*Observed behaviour:* :eyes: :broken_heart:
Nothing happens

*Expected behaviour:* :heart: :smile:
Spacemacs should load the typescript definition into a new buffer

#### System Info :computer:
- OS: darwin
- Emacs: 26.1
- Spacemacs: 0.200.13
- Spacemacs branch: master (rev. c7a103a77)
- Graphic display: nil
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(ansible asm auto-completion c-c   chrome colors common-lisp cscope csharp csv deft django docker elfeed emacs-lisp extra-langs fasd finance floobits games git github go gtags helm html imenu-list java javascript lua markdown nginx nlinum org pandoc pdf-tools php plantuml prodigy python ranger react restclient ruby ruby-on-rails search-engine semantic shell shell-scripts speed-reading spell-checking sql syntax-checking systemd terraform themes-megapack twitter typescript typography version-control wakatime windows-scripts yaml ycmd)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB THREADS

#### Backtrace :paw_prints:
I tried this in debug mode and there are no errors. Simply put, nothing happens.

**Comment by codejamninja** [comment]
I dug a little deeper. Basically `typescript/jump-to-type-def` calls `tide-jump-to-definition`.

I tried running `tide-jump-to-definition` and it works great. It only fails when using `typescript/jump-to-type-def`.

**Comment by codejamninja** [comment]
I'm not sure why `true` is being passed to the `tide-jump-to-definition` function. When I don't pass any args to it, it fixes this bug.

```lisp
      (defun typescript/jump-to-type-def()
        (interactive)
        (tide-jump-to-definition t)) ;; removing the t arg fixes the bug
```

https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Blang/typescript/packages.el#L54

**Comment by codejamninja** [comment]
The bug is in a different location in the `develop` branch, but it's the same bug.

https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Blang/typescript/funcs.el#L155

**Comment by codejamninja** [comment]
I created a pull request to fix this at #11629
